sdk/src/main/scala/coop/rchain/sdk/dag/merging/DagMergingLogic.scala

```diff
@@ -104,9 +104,10 @@ object DagMergingLogic {
       conflicts: (D, D) => Boolean,
       depends: (D, D) => Boolean
   ): (Map[D, Set[D]], Map[D, Set[D]]) = {
-    val conflictsMap = computeConflictsMap(conflictSet, finalSet, conflicts) ++
+    val conflictsMap = computeConflictsMap(conflictSet, finalSet, conflicts) |+|
```

Comment:
Here combination instead of plain sum should be used, as message in conflict set might conflict with something in conflict set and final set both.